core/include/tee/tee_svc.h

```diff
@@ -98,12 +98,12 @@ static inline uint32_t tee_svc_kaddr_to_uref(void *kaddr)
 	return (vaddr_t)kaddr - tee_svc_uref_base;
 }
 
-static inline vaddr_t tee_svc_uref_to_vaddr(uint32_t uref)
+static inline vaddr_t tee_svc_uref_to_vaddr(unsigned long uref)
```

Comment:
A `uref` is always supposed to be in the range of a `uint32_t`, see `tee_svc_kaddr_to_uref()` for instance.